notes on review of contributions (#43)
LSSTDESC/DC2-analysis: README.md

New scripts and notebooks can be added in the [contributed directory](https://github.com/LSSTDESC/DC2-analysis/tree/master/contributed).  The [README](https://github.com/LSSTDESC/DC2-analysis/tree/master/contributed/README.md) has a full description of how to do so.

## Notebook review

There are two reasons why one might add a notebook to the repository:

> 1) Adding a enerally applicable notebook that you think should be added to the master set of notebooks for DC2 analysis (we imagine the DC2 team will be adding most of these). These will be reviewed by another DC2 analysis team member so the fix/contribution can be added to the tutorial set.

Reviewer: ```suggestion
1) Adding a generally applicable notebook that you think should be added to the master set of notebooks for DC2 analysis (such as a tutorial or a validation notebook; we imagine the DC2 team will be adding most of these). These will be reviewed by another DC2 team member.
```

> 2) You have a notebook for your particular science case that you'd like to add (it could combine some tasks from the existing notebooks, or do something totally different etc.) For these notebooks, we will engage the analysis teams for input.

Reviewer: ```suggestion
2) You have a notebook for your particular science case that you'd like to add (usually under `contributed`; the notebook could combine some tasks from the existing notebooks, or do something totally different etc). For these notebooks, we will engage the analysis teams for input.
```

Some suggestions for adding new notebooks (e.g. contribution type 2 above). The code should:

> a) run without issues, and 

Reviewer 2: ```suggestion
a) run without issues; 
```

> b) have some texts on what the notebook is trying to do 

Reviewer 2: ```suggestion
b) have some explanatory text about what the notebook is trying to do; 
```

> c) use recommended packages/instructions whenever possible (e.g., not skipping API to access underlying files) and if not, clearly document their atypical/unrecommended usage

Reviewer 2: ```suggestion
c) use recommended packages/instructions whenever possible (e.g., not skipping API to access underlying files) and if not, clearly document their atypical/unrecommended usage; and
```

> d) reasonable coding style/practice.

Reviewer 2: ```suggestion
d) follow reasonable coding style/practices (for some examples, see the existing tutorial notebooks; or see the [DESC Coding Guidelines](https://docs.google.com/document/d/1v54bVQI2NejK2UqACDnGXj1t6IGFgY3Uc1R7iV2uLpY/edit?usp=sharing)).
```
